Docs 6745 export mule app mc - download complete mule app (#125)
mulesoft/docs-design-center: modules/ROOT/pages/fd-share-to-studio.adoc

:imagesdir: ../assets/images

> 

Reviewer: Please consider or apply the edits to the previous .adoc file to this one.
Much of the test is the same.
I have copyedited only the changed text.

> Perhaps when you are creating a Mule application you need help from someone in your Anypoint-Platform organization,

Reviewer: ```suggestion
When designing your Mule application, you might want help from someone else in your Anypoint Platform organization,
```

Author: Fixed.

> such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can

Reviewer: ```suggestion
such as a developer, who works in Anypoint Studio. You can share your project with that person by downloading it as a `.jar` file and placing the file in a location from which the other person can
```

Author: using your previous edits. Fixed.

> import it into Studio.

Reviewer: ```suggestion
import it into Studio. To do so, you use *Export project sources* to share just the source files (including all the connector credentials), without all of the other Mule artifacts.
```

Author: Fixed.

You'll need to use *Export project sources* to share just your project source files (including all the connector credentials)

> without all of the other artifacts that are required by the Mule runtime.

Reviewer: See above

Author: Fixed.

> As the other person works on the project, the changes are applied to your project online in Flow Designer.

Reviewer 2: @mcherem this line is confusing to me. there is no link between what's happening offline in studio and the flow designer project online. Also because Studio offers more capabilities today - it isn't always true that the user can then re-import the app back into FD after some changes have been made in Studio land

Author: Fixed.

As the other person works on the project, they can take advantage of the advanced features of Studio.

> When they are finished, they can choose to deploy the project from Studio.

Reviewer: Consider adding something about the advanced features of Studio into a previous sentence.
Not sure we want the second sentence here: does this mean that the user-audience does NOT deploy, or cannot deploy?

Author: These are features that are not in FD. It's a lot to mention here. Skipping for now.

. In Flow Designer, ensure that your project is free of errors.
. Export your project by clicking the dots next to the Test button and select *Export* from the list.
. Select *Export project sources* to share just your project source files, and click *Export*.

+
A `.jar` file that contains your project source files, including all the connector credentials, downloads to the default location on your computer.
+
// Place the .jar file in a location that the other person can access.

. Have the person who works in Studio follow these steps:
.. Enable Studio to access your organization in Anypoint Platform by selecting *Preferences* > *Anypoint Studio* > *Authentication*.
.. Import the `.jar` file by selecting *File* > *Import* and then selecting *Anypoint Studio* > *Packaged Mule Application (.jar)*.
+
Studio opens the project.

> The person who works in Studio can work on the project. 

Reviewer: and deploy it? explain, per previous comment

Author: Fixed.

> The person who works in Studio can work on the project, and all changes are made automatically to the project in Design Center.

Reviewer 2: same comment here - this is not possible today

Author: Fixed.

+
Studio opens the project.

> The person who works in Studio can work on the project. When the work in Studio is completed, ensure that the project is deleted from Studio.

Reviewer 2: suggest we just kill this line 
> When the work in Studio is completed, ensure that the project is deleted from Studio. If you need help again later, repeat the procedure to share your project.

looks good otherwise

Author: Fixed.
